daml-foundations/daml-ghc/daml-prim-src/GHC/Base.daml

```diff
@@ -57,12 +57,23 @@ infixr 9 .
 const                   : a -> b -> a
 const x _               =  x
 
+-- | A `Functor` is a typeclass for things that can be mapped over (using
+-- its `fmap` function. Examples include `Maybe` and `[]`.
+
+-- For a detailed explanation, see 
+-- http://learnyouahaskell.com/making-our-own-types-and-typeclasses#the-functor-typeclass
+
 class Functor f where
```

Comment:
I think you need to remove the blank lines, replacing them with `--`, or the docs won't show up.

Reply:
Ok, good to know! I meant to generate the docs to test this and forgot - will do today.